src/__deprecated__/components/radio-button/radio-button.scss

```diff
@@ -19,6 +20,7 @@
 
   &.common-input--has-field-help {
     .common-input__label {
+      vertical-align: unset;
```

Comment:
Just wondering whether it's OK to use `unset` ? It's not supported by IE:

* https://developer.mozilla.org/en-US/docs/Web/CSS/unset#Browser_compatibility
* https://caniuse.com/#feat=css-unset-value

Reply:
Ah yeah very good point, I'll make a safer change

Reply:
Thanks. `unset` is used in a couple of other places too:

* https://github.com/Sage/carbon/blob/5efe0b7/src/__deprecated__/components/checkbox/checkbox.scss#L26
* https://github.com/Sage/carbon/blob/5efe0b7/src/components/menu/menu-item/menu-item.scss#L158

Reply:
Yeah I'll fix the checkbox style, but I'm going to leave the menu-item one (I'll make a jira ticket) as it's not in scope for this and makes the QA more fiddly on this PR 😅

Reply:
OK! :ok_hand: